lib/src/Base/Stat/SampleImplementation.cxx

```diff
@@ -1573,6 +1573,32 @@ void SampleImplementation::sortUniqueInPlace()
   if (last + 1 < size_) erase(last + 1, size_);
 }
 
+Indices SampleImplementation::argsort(Bool isIncreasing) const
+{
+  Collection< std::pair<Point, UnsignedInteger> > pointsPairs(size_);
+  for (UnsignedInteger i = 0; i < size_; ++i)
```

Comment:
You should swap the test and the for loop. It will save you many tests even if the drawback is to write two for loops.

Reply:
Of course. Done.